addons/docs/src/frameworks/common/inferArgTypes.ts

```diff
@@ -16,7 +16,7 @@ const inferType = (value?: any): SBType => {
   if (Array.isArray(value)) {
     const childType: SBType =
       value.length > 0 ? inferType(value[0]) : { name: 'other', value: 'unknown' };
```

Comment:
So we don’t check every value in the array has the same type?

Reply:
Not currently. WDYT is a good behavior in the case that they don't match? union type? error? any?